backend/app/modules/user/antisybil/service/guest_list.py

```diff
@@ -0,0 +1,470 @@
+guest_list = [
```

Comment:
please move it to backend/app/constants.py and you can make it as set in advance, i.e ```guest_list = set(["0x16f3f2f0ba34973937A1ebb989a295Ca106b67C7", ...])```

Reply:
+1